components/CreateTodo.js

```diff
@@ -0,0 +1,118 @@
+
+import Toast from "./Toast.js";
+
+const CreateTodo = () => {
+  const buttonCreate = document.querySelector('#buttonCreate');
+  const createTodo = document.querySelector('.create-todo');
+  const cancelTodoBtn = document.querySelector('#cancelTodo');
+  const textArea = document.querySelector('.area');
+
+  const addTodoBtn = document.querySelector('#addTodo');
+  const todoWrapper = document.querySelector('.todo-wrapper');
+
+  const openCreateTodo = () => {
+    createTodo.classList.add('open-create-todo');
+    textArea.focus();
+  }
+
+  const cancelTodo = () => {
+    createTodo.classList.remove('open-create-todo');
+    textArea.value = '';
+  }
+
+  buttonCreate.addEventListener('click', openCreateTodo);
+  cancelTodoBtn.addEventListener('click', cancelTodo);
+
+  const emptyNotice = document.createElement('h2');
+
+  if(todoWrapper.innerHTML === '') {
+    emptyNotice.innerText = 'Todo list is empty';
+    todoWrapper.appendChild(emptyNotice)
+  }
+
+  const addTodo = () => {
+    const messageConfirmation = Toast('Please enter todo', 'error');
+
+    if(!textArea.value) {
+      todoWrapper.appendChild(messageConfirmation);
+      setTimeout(() => {
+        messageConfirmation.remove();
+      }, 2000)
+    }
```

Comment:
If no textArea value is provided we have to return early
```suggestion
    if(!textArea.value) {
      todoWrapper.appendChild(messageConfirmation);
      setTimeout(() => {
        messageConfirmation.remove();
      }, 2000)
      
      return
    }
```